app/src/main/java/protect/card_locker/ManageGroupActivityInGroupState.java

```diff
@@ -0,0 +1,58 @@
+package protect.card_locker;
+
+import android.os.Parcel;
+import android.os.Parcelable;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class ManageGroupActivityInGroupState implements Parcelable {
```

Comment:
What is this class for?

Reply:
during a screen rotation, before onDestroy is called, onSaveInstanceState is invoked and I can save Parcelables for when onCreate is invoked again
I use this class to save and restore mAdapterState so that user selections persists during things like screen rotation

Reply:
going to use a hash map to integer array function instead inside the activity